resources/views/components/game/link-buttons/index.blade.php

```diff
@@ -73,17 +72,6 @@
         @endcan
     @endif
 
-    @if (in_array('manage-hashes', $allowedLinks))
-        @can('manage', App\Models\GameHash::class)
-            <x-game.link-buttons.game-link-button
-                icon="💾"
-                :href="route('game.hash.manage', ['game' => $game])"
```

Comment:
Not sure where to put this note, but the Manage Hashes buttons still exists on the Supported Game Files page and does correctly link to the filament page, so this didn't remove that.

Conversely, the Manage Hashes link in the Dev box on the game page is now broken.